source/frontend/src/components/maps/leaflet/Legend/Legend.tsx

```diff
@@ -17,6 +17,10 @@ export const Legend = () => {
         pin: require('assets/images/pins/land-poi.svg').default,
         label: 'Property of Interest',
       },
+      {
+        pin: require('assets/images/pins/land-lease.svg').default,
+        label: 'Payable lease/license',
+      },
```

Comment:
not a big deal but I see four (4) images added on this PR - 2 svgs and 2 pngs but I only see the svgs being used in code. Are the pngs used or could they be safely removed?

Reply:
My guess is that they could be removed if not used. But I did not check if the other marker images have the same usage.